src/vs/workbench/contrib/welcome/telemetryOptOut/browser/telemetryOptOut.ts

```diff
@@ -139,7 +139,6 @@ export abstract class AbstractTelemetryOptOut implements IWorkbenchContribution
 						run: () => {
 							logTelemetry(true);
 							this.configurationService.updateValue('telemetry.enableTelemetry', false);
```

Comment:
@deepak1556 this needs to write to the `argv.json` to disable crash reporter, no? How would we otherwise preserve the behaviour we have today?

Reply:
this is still pending, how would you suggest we do it ? Is there a way to read the telemetry value on startup ?

Reply:
No, there is not. I would consider to use `IJSONEditingService` for this purpose, it supports comments and allows to preserve the formatting. We use this service to programmatically write into setting files too.